lib/smart_answer_flows/calculate-agricultural-holiday-entitlement.rb

```diff
@@ -35,11 +35,9 @@ def define
         option "2-days"
         option "1-day"
 
-        calculate :days_worked_per_week do |response|
-          # XXX: this is a bit nasty and takes advantage of the fact that
-          # to_i only looks for the very first integer
-          response.to_i
-        end
+        # XXX: this is a bit nasty and takes advantage of the fact that
+        # to_i only looks for the very first integer in the string response
+        calculate :days_worked_per_week, &:to_i
```

Comment:
This looks odd to me but maybe it's not such a big problem as hopefully these `calculate` blocks won't be around for _too_ much longer.